### PR TITLE
Fix scroll

### PR DESCRIPTION
**Pull request by adiletelf**
When you scroll the visual horizontally the expected behavior is for the header to stick in the same position, but it started to move left on the scroll, making it invisible for the user.

Now, with the scroll event the header is moving accordingly and is always visible.

### CODE REVIEW

File: src/timeLineSettingsModel.ts [hunk 1]
```diff
@@ -285,20 +236,73 @@ export class CellsSettingsCard extends Card {
...
-        this.strokeSelected,
-        this.fillUnselected,
-        this.strokeUnselected,
-        this.strokeWidth,
-        this.gapWidth,
-        this.enableManualSizing,
-        this.width,
-        this.height,
-    ];
+    groups = [this.cellsGeneralGroup, this.selectedCellsGroup, this.unselectedCellsGroup];
```
Comment (Demonkratiy) [inline src/timeLineSettingsModel.ts]: better to keep naming pattern going from one object to another: cells -> selected -> group = cellsSelectedGroup, not selectedCellsGroup

File: stringResources/en-US/resources.resjson [hunk 2]
```diff
@@ -62,8 +64,7 @@
 	"Visual_DisplayMonths": "Display months",
 	"Visual_DisplayWeeks": "Display weeks",
 	"Visual_DisplayDays": "Display days",
-	"Visual_Cursor": "Cursor",
-	"Visual_CursorColor": "Cursor color",
+	"Visual_PointerEdge": "Pointer edge",
```
Comment (Demonkratiy) [inline stringResources/en-US/resources.resjson]: Seems we don't plan to use `Visual_PointerEdge` anymore?

Comment (adiletelf) [reply stringResources/en-US/resources.resjson]: You're certainly right.

File: src/timeLineSettingsModel.ts (hunk 1 above)
Comment (adiletelf) [reply src/timeLineSettingsModel.ts]: Renamed.

**Review by Demonkratiy — APPROVED** [review]

**Pull request merged**